site/docs/get-started/docker.md

```diff
@@ -13,13 +13,13 @@ is available).
 Pull the ManageIQ docker image:
 
 ```bash
-$ sudo docker pull manageiq/manageiq:latest-darga
+$ sudo docker pull manageiq/manageiq:latest-euwe
```

Comment:
`latest-darga` and `latest-euwe` are nightly builds. A release tag (e.g. `euwe-1`) should be used instead?